app/livechat/lib/LivechatInquiry.js

```diff
@@ -100,6 +142,29 @@ if (Meteor.isServer) {
 
 			return this.update(query, update);
 		}
+
+		async getCurrentSortedQueue({ _id, department }) {
```

Comment:
Should we name these methods as `Async` like `getCurrentSortedQueueAsync` to prevent misusage?

Reply:
Sure! 
Done.